fixtures/ts/eslint/editorconfig

```diff
@@ -0,0 +1,7 @@
+[*]
+charset = utf-8
+indent_size = 2
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
duplication of the original `.editorconfig`. Would be nice if we can reuse from the same file, instead of maintaining it in 2 places (unless that creates a lot more headaches)

Reply:
Since this file is exactly the same I agree. I did this as well with `.eslintignore` (only 1 line difference), I figure one irrelevant ignore line won't hurt folks using JS

Reply:
Agreed 👍